entrypoint.sh

```diff
@@ -104,6 +108,9 @@ run_client() {
     fi
 }
 
+echo "Update using wineboot. See $wine_logfile_name for logs."
```

Comment:
Since the boot is still not instantaneous, it would be helpful to also say something like `Please wait ~60s` or something to let users know that a pause here is expected.